src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/ContainingSyntaxContext.cs

```diff
@@ -12,8 +12,12 @@
 
 namespace Microsoft.Interop
 {
-    public readonly record struct ContainingSyntax(SyntaxTokenList Modifiers, SyntaxKind TypeKind, SyntaxToken Identifier, TypeParameterListSyntax? TypeParameters)
+    public readonly record struct ContainingSyntax(SyntaxTokenList modifiers, SyntaxKind TypeKind, SyntaxToken identifier, TypeParameterListSyntax? TypeParameters)
```

Comment:
Maybe we should make this a regular struct and not a record since we already override equals, then we won't have both `modifiers` and `Modifiers`